test/safe-esm/teenytest-proxy.js

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+// This file is needed because of bugs in ESM in Node.js (around loaders):
+// https://github.com/nodejs/node/issues/33226
+// https://github.com/nodejs/node/issues/33303
+//
+// Once these bugs are solved, we can go back to running teenytest regularly for ESM tests
+
+require('../../node_modules/.bin/teenytest')
```

Comment:
Mind setting a todo for yourself a few months from now to come back and revisit this?